src/dbus_api.rs

```diff
@@ -43,19 +43,21 @@ pub enum DeferredAction {
 
 #[derive(Debug, Clone)]
 pub struct DbusContext {
+    pub action_list: Rc<RefCell<Vec<DeferredAction>>>,
```

Comment:
My first reaction to alphabetizing struct members is negative. This doesn't help readability. I have a hard time articulating a better policy other than "put them in the order they should be in" or "it doesn't matter" but I do think there are reasons to order members a certain way, and a policy to alphabetize them is counterproductive.

This commit also mixes a substantive change with the ordering change which is stylistic, so it would've been nice for them to be in separate commits.